_data/banking.yml

```
@@ -319,6 +319,15 @@ websites:
     facebook: CDCFederalCreditUnion
     img: cdcfcu.png
 
+  - name: Cembra Money Bank
+    url: https://www.cembra.ch/
+    tfa:
+      - sms
+    img: cembra-money-bank.png
+    regions:
+      - ch
+    doc: https://faq.cembra.ch/de/videos/login-mit-mtan/
```

Although it's not strictly necessary, we try to keep the lines for an entry in a certain order, so this is just a bit of shuffling around:
```suggestion
    img: cembra-money-bank.png
    tfa:
      - sms
    doc: https://faq.cembra.ch/de/videos/login-mit-mtan/
    regions:
      - ch
```

---

@phallobst order changed

---

You could have just used "Commit suggestion" to apply the suggested changes, no need to do it manually.